src/pages/trading-specification/components/_elements.tsx

```diff
@@ -233,21 +232,21 @@ type TTableCellGroup = {
 }
 export const TableCellGroup = ({ data, market }: TTableCellGroup) => {
     const { is_row } = useRegion()
-    const { is_mobile_or_tablet } = useBreakpoints()
     const { symbol, instrument: text, dl_icon, swf_icon } = data
     const [show_popUp, setShowPopUp] = useState(false)
     const [popup_type, setPopupType] = useState<TPopupType>()
     const [scrollY, setScrollY] = useState<number>()
 
     useEffect(() => {
-        if (is_mobile_or_tablet) {
-            document.body.style.position = show_popUp ? 'fixed' : ''
-            document.body.style.width = show_popUp ? '100%' : ''
-            if (!show_popUp && scrollY) {
-                window.scrollTo({ top: scrollY, behavior: 'smooth' })
-            }
+        document.body.style.position = show_popUp ? 'fixed' : ''
+        document.body.style.width = show_popUp ? '100%' : ''
+        if (!show_popUp && scrollY) {
```

Comment:
Can we double check if desktop is not affected after the condition is removed?

Reply:
QA requested for consistent behaviour for both Desktop and Mobile

Reply:
Ok I see.